frontend/packages/component-library-styles/mixins.scss

```
@@ -170,3 +171,9 @@
   line-height: $line-height;
   width: calc(#{$font-size} * #{$line-height});
 }
+
+// Border styles
+@mixin regular-border-styles {
+  border: 1px solid var(--borders-neutral-primary);
+  border-radius: variables.$regular-border-radius;
+}
```

I'm not sure how regular these styles are 😁  Right now   `border: 1px solid var(--borders-neutral-primary);` line is only used in `Accordion`, `ActionBlock`, `Tabs`, `Image`.

I agree thought that this is a common pattern. What do you think if we make it like this instead?
```suggestion
// Border styles
@mixin border-styles($border-color: var(--borders-neutral-primary), $border-width: 1px, $border-style: solid, $border-radius: variables.$regular-border-radius) {
  border: $border-width $border-style $border-color;
  border-radius: $border-radius;
}
```
It should save us time by having default values and makes it easier to style borders for us by only adding passing arguments that we need (usually just border-width and color)

---

Oh yeah I like this! 🙌